### ISSUE TITLE
Contact us link

### ISSUE THREAD

**Issue by gtvj** [nationalarchives/ds-wagtail#170]
On page 20 of the report DAC describe an issue relating to the "Contact us" link being duplicated. No specific WCAG failure is described. 

We can fix this by giving this link unique and descriptive text. DAC suggest "Contact us via email".

### Why wasn't this issue identified by in-house testing
I believe we'd asked DAC not to test this page (as it's not part of the service). It also isn't described as a WCAG failure.
